--- Finder.kt
+++ Finder.kt
@@ -142,10 +142,10 @@
      * @return 見つかったプロパティを [KtProperty] として返します。見つからない場合は null を返します。
      */
     fun findKtProperty(qualifiedPropertyName: String): KtProperty? {
-        val targetFqName = FqName(qualifiedPropertyName)
-        return findKtClass(targetFqName.parent().asString())?.let { ktClass ->
+        val propertyName = qualifiedPropertyName.substringAfterLast(".")
+        return findKtClass(qualifiedPropertyName.substringBeforeLast("."))?.let { ktClass ->
             val properties = ktClass.getProperties()
-            return properties.firstOrNull { it.fqName == targetFqName }
+            return properties.firstOrNull { it.name == propertyName }
         }
     }
 